### DISCUSSION TITLE
ONVIF H264

### DISCUSSION THREAD

**Discussion by gmcaesar** [roleoroleo/sonoff-hack#42] [Q&A]
When I add this to my Home Assistant it doesn't receive the H264 encoded signal - there is no audio and chromecast will not play the stream, do i need to choose another profile? or is there an option somewhere I am missing?

**Comment by roleoroleo** [comment]
If I'm not wrong HA doesn't support audio in camera streams. But you should receive the h264 video.
What's your configuration in ha?
How do you play the stream to chromecast?

**Reply by gmcaesar** [reply]
When I brought this issue to the onvif integration got this was the response:
https://github.com/home-assistant/core/issues/46719#issuecomment-789760834

**Reply by roleoroleo** [reply]
Yes the only audio supported by ha is aac.
And this cam uses alaw pcm.

**Reply by gmcaesar** [reply]
I was hoping to use the sound level as a trigger for automations, is this something you might be able to add?

**Reply by roleoroleo** [reply]
Not in onvif, sorry.
I spent a lot of days to add events to onvif but I couldn't fix the memory leak problem.
No way at the moment.

**Reply by gmcaesar** [reply]
Is there another way? I'm not tied to onvif
